liberty-ls/src/main/java/io/openliberty/tools/langserver/utils/ServerPropertyValues.java

```diff
@@ -0,0 +1,54 @@
+package io.openliberty.tools.langserver.utils;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+
+import io.openliberty.tools.langserver.ls.LibertyTextDocument;
+
+/**
+ * Maps property keys with their list of valid values for server.env and bootstrap.properties
+ */
+public class ServerPropertyValues {
```

Comment:
What about properties that have a certain data type, like it should be an integer? Is there anything we will do to flag incorrect values for those?

Reply:
Ok, if we don't have an issue open for validating values, please go ahead and open one.